libs/platform/src/lib/components/dynamic-page/dynamic-page-footer/dynamic-page-footer.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component, ChangeDetectionStrategy, forwardRef } from '@angular/core';
+
+@Component({
+    selector: 'fdp-dynamic-page-footer',
+    template: '<div style="position:relative; margin-top: 3rem"><ng-content></ng-content></div>',
```

Comment:
Could you create some class for those styles? Inline styling is pretty hard to overwrite by user. 
Also let's remove encapsulation, by adding `ViewEncapsulation.None`

Reply:
these inline styles are no longer required, hence removed. Also, added `ViewEncapsulation.None`